addons/website_slides/models/slide_channel.py

```diff
@@ -1024,16 +1022,14 @@ def _remove_membership(self, partner_ids):
         if not partner_ids:
             raise ValueError("Do not use this method with an empty partner_id recordset")
 
-        removed_channel_partner_domain = []
-        for channel in self:
-            removed_channel_partner_domain = expression.OR([
-                removed_channel_partner_domain,
-                [('partner_id', 'in', partner_ids),
-                 ('channel_id', '=', channel.id)]
-            ])
+        removed_channel_partner_domain = expression.OR([
+            [('partner_id', 'in', partner_ids),
+             ('channel_id', '=', channel.id)]
+            for channel in self
+        ])
 
         self.message_unsubscribe(partner_ids=partner_ids)
-        if removed_channel_partner_domain:
+        if removed_channel_partner_domain != expression.FALSE_DOMAIN:
```

Comment:
Same as above, this is equivalent to
```suggestion
        if self:
```